lib/components/TinyMCE.js

```diff
@@ -51,22 +52,31 @@ const TinyMCE = React.createClass({
   },
 
   componentDidMount() {
-    const config = clone(this.props.config);
-    this._init(config);
+    this._init(this.props.config, this.props.content);
   },
 
   componentWillReceiveProps(nextProps) {
-    if (!isEqual(this.props.config, nextProps.config)) {
-      this._init(nextProps.config, nextProps.content);
-    }
-    if (!isEqual(this.props.id, nextProps.id)) {
+    if (!isEqual(this.props.config, nextProps.config) || !isEqual(this.props.id, nextProps.id)) {
       this.id = nextProps.id;
+      this._init(clone(nextProps.config), nextProps.content);
+      return;
+    }
+    else if (!isEqual(this.props.content, nextProps.content)) {
+      this._updateContent(nextProps.content);
     }
+
+    // TODO: This will fix problems with keeping internal state, but causes problems with cursor placement
```

Comment:
Should we add a note here that we've implemented this elsewhere? Or adjust this TODO somehow?